src/sources/demo_logs.rs

```diff
@@ -105,26 +105,26 @@ pub enum OutputFormat {
         lines: Vec<String>,
     },
 
-    /// Randomly generated logs in [Apache common][apache_common] format.
-    /// [apache_common]: https://httpd.apache.org/docs/current/logs.html#common
+    /// Randomly generated logs in
+    /// [Apache common](https://httpd.apache.org/docs/current/logs.html#common) format.
     ApacheCommon,
 
-    /// Randomly generated logs in [Apache error][apache_error] format.
-    /// [apache_error]: https://httpd.apache.org/docs/current/logs.html#errorlog
+    /// Randomly generated logs in
+    /// [Apache error](https://httpd.apache.org/docs/current/logs.html#errorlog) format.
     ApacheError,
 
-    /// Randomly generated logs in Syslog format ([RFC 5424][syslog_5424]).
-    /// [syslog_5424]: https://tools.ietf.org/html/rfc5424
+    /// Randomly generated logs in Syslog format
+    /// ([RFC 5424](https://tools.ietf.org/html/rfc5424)).
     #[serde(alias = "rfc5424")]
     Syslog,
 
-    /// Randomly generated logs in Syslog format ([RFC 3164][syslog_3164]).
-    /// [syslog_3164]: https://tools.ietf.org/html/rfc3164
+    /// Randomly generated logs in Syslog format
+    /// ([RFC 3164](https://tools.ietf.org/html/rfc3164)).
     #[serde(alias = "rfc3164")]
     BsdSyslog,
 
-    /// Randomly generated HTTP server logs in [JSON][json] format.
-    /// [json]: https://en.wikipedia.org/wiki/JSON
+    /// Randomly generated HTTP server logs in
+    /// [JSON](https://en.wikipedia.org/wiki/JSON) format.
```

Comment:
I'd prefer to have these as-is, they were and with your changes broken on the use facing docs. There just needs to be line breaks between the body and reference.